openslides_backend/migrations/migrations/0054_remove_forwarding_user.py

```diff
@@ -0,0 +1,28 @@
+from datastore.migrations import BaseModelMigration
+from datastore.shared.util import fqid_from_collection_and_id
+from datastore.writer.core import BaseRequestEvent, RequestUpdateEvent
+
+
+class Migration(BaseModelMigration):
+    """
+    This migration removes the forwarding user relation
+    """
+
+    target_migration_index = 55
+
+    def migrate_models(self) -> list[BaseRequestEvent] | None:
+        events: list[BaseRequestEvent] = []
+        for collection, field in [
```

Comment:
no need to put the collection into loop head and where is the collection from?

Reply:
This loop goes through a list of tuples in a (collection, field) format.
That's where the collection comes from.

This is the case since this migration needs the exact same functionality for two different fields in two different collections. (i.e. "Delete the field from all instances that have it")

This was the least redundant way to write that code.